erpnext/buying/doctype/purchase_order_item/purchase_order_item.json

```diff
@@ -726,13 +727,19 @@
    "fieldname": "more_info_section_break",
    "fieldtype": "Section Break",
    "label": "More Information"
+  },
+  {
+   "fieldname": "stock_uom_rate",
+   "fieldtype": "Currency",
+   "label": "Rate (Stock UOM)",
+   "read_only": 1
```

Comment:
should this be visible if uom = stock uom. Then it will just show the same value. Hide if both uom fields are the same maybe ? (display depends on)